isolate_example/lib/page_two.dart

```diff
@@ -35,6 +35,13 @@ class InfiniteProcessPage extends StatelessWidget {
     return Column(
       mainAxisSize: MainAxisSize.min,
       children: [
+        Container(
+          child: Text(
+            'Summation Results',
+            style: TextStyle(fontSize: 25),
+          ),
+          padding: new EdgeInsets.all(8),
```

Comment:
No need to use a `Container` if all you're looking for is Padding. Just wrap the `Text` in a `Padding` widget.